pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -246,7 +246,7 @@ public static FunctionInfo getFunctionInfo(String name, int numArguments) {
   }
 
   public static String canonicalize(String name) {
-    return StringUtils.remove(name, '_').toLowerCase();
+    return StringUtils.remove(StringUtils.remove(name, '_'), ' ').toLowerCase();
```

Comment:
This brings overhead to all functions. Can we modify the name of the operator to avoid this step?

Reply:
I can try to take a look tomorrow, but IIRC I tried that and it ended up producing parsing issues. On the other hand, ni don't think the cost is too heavy. In the worse case scenario I can rewrite this code to do the same in a single copy and a single iteration over the name of the function

Reply:
This is also causing inconsistency between `FunctionRegistry` and `TransformFunctionFactory`. Ideally I want to keep them the same

Reply:
I didn't find why yet, but if I change the name to is_null (and is_not_null) and remove the ones called "is null" or "is not null", queries fail with:

```
From line 5, column 5 to line 5, column 22: No match found for function signature IS NOT NULL
```